.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+exclude: ^(src/blib2to3/|profiling/|unit_tests/data/)
+
+repos:
+  - repo: https://github.com/pycqa/pylint
+    rev: pylint-2.6.0
+    hooks:
+      - id: pylint
+  - repo: git://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-merge-conflict
+      - id: check-symlinks
+      - id: check-yaml
+        args: [--unsafe]
+      - id: debug-statements
+      - id: detect-private-key
+      - id: end-of-file-fixer
+  - repo: "https://github.com/psf/black"
+    rev: "20.8b1"
+    hooks:
+      - id: "black"
+        name: "Format code (black)"
+        language_version: "python3"
```

Comment:
For the time being I would not include this in the PR, we do not know if we want to use black yet and it is quite low in our backlog. See #67 #68 [OAMG-3343](https://issues.redhat.com/browse/OAMG-3343).

Reply:
I see black and isort as something like low-hanging fruits which has an excellent positive effect on the developer productivity. I believe the whole  this MR is not driven by the priorities in the backlog, but if we're adding pre-commit, then it will be a mistake to skip black and isort.